### PR TITLE
Backport same PRs as done in bitcoin#11447

### PR DESCRIPTION
**Pull request by codablock**
This does the same as #3110, but for bitcoin#11447

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -109,6 +109,9 @@ See detailed [set of changes](https://github.com/dashpay/dash/compare/v0.14.0.2.
 - [`4286dde49`](https://github.com/dashpay/dash/commit/4286dde49) Remove support for InstantSend locked gobject collaterals (#3019)
 - [`788d42dbc`](https://github.com/dashpay/dash/commit/788d42dbc) Bump version to 0.14.0.3 and copy release notes (#3053)
 
+- `dumpwallet` no longer allows overwriting files. This is a security measure
```
Comment (UdjinM6) [inline doc/release-notes.md]: should drop this part

Comment (codablock) [reply doc/release-notes.md]: done

File: src/rpc/client.cpp [hunk 2]
```diff
@@ -142,8 +142,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "keypoolrefill", 0, "newsize" },
     { "getrawmempool", 0, "verbose" },
     { "estimatefee", 0, "nblocks" },
-    { "estimatesmartfee", 0, "nblocks" },
-    { "estimaterawfee", 0, "nblocks" },
+    { "estimatesmartfee", 0, "conf_target" },
+    { "estimaterawfee", 0, "conf_target" },
     { "estimaterawfee", 1, "threshold" },
     { "prioritisetransaction", 1, "fee_delta" },
     { "setban", 2, "bantime" },
```
Comment (PastaPastaPasta) [inline src/rpc/client.cpp]: missing changes in mining.cpp 

`
     { "util",               "estimatesmartfee",       &estimatesmartfee,       {"nblocks", "estimate_mode"} }, 
` ->
`
     { "util",               "estimatesmartfee",       &estimatesmartfee,       {"conf_target", "estimate_mode"} }, 
`

Comment (UdjinM6) [reply src/rpc/client.cpp]: That's because this was already done in #3096 when we backported 10707.

**Pull request merged**